### PR TITLE
payable vouchers tests and withdrawEther function

### PR DESCRIPTION
**Pull request by ZzzzHui**

### CODE REVIEW

**Comment by ZzzzHui** [comment]
I restored the `withdrawEther` function, but I'm in favor of removing it sometime soon. Because its functionality is redundant and more withdrawal methods mean more opportunities for vulnerabilities.

**Comment by guidanoli** [comment]
> Because its functionality is redundant and more withdrawal methods mean more opportunities for vulnerabilities.

We can mark this function as deprecated and later remove it. :-)

File: onchain/rollups/test/foundry/dapp/CartesiDApp.t.sol [hunk 1]
```diff
@@ -570,7 +570,7 @@ contract CartesiDAppTest is TestBase {
         assertEq(address(simpleContract).balance, transferAmount);
     }
 
-    function testWithdrawEtherToContractWithPayload(
+    function testWithdrawEtherToContractWithRandomPayload(
```
Comment (guidanoli) [inline onchain/rollups/test/foundry/dapp/CartesiDApp.t.sol:573]: When I read "random" payload, my first thought is that it is a fuzzy variable.
Instead, it is a hard-coded, seemingly random payload.
So, we could say it is just a wrong payload.

Comment (ZzzzHui) [reply onchain/rollups/test/foundry/dapp/CartesiDApp.t.sol:573]: Make sense 👍

**Comment by guidanoli** [comment]
Can we close this, @ZzzzHui ?

**Pull request closed without merging**